### DISCUSSION TITLE
Off-topic question: add google colab units

### DISCUSSION THREAD

**Discussion by darth2602** [TheLastBen/fast-stable-diffusion#2034] [Q&A]
Bonjour, question hors sujet pour les utilisateurs de google colab :

j'ai un abonnement Google Colab Pro et je suis arrivé au bout de mes unités, on me demande de racheter des unité
et on me donne 3 choix : 

1) - Pay as you go 
2) - google colab pro 
3) - google colab pro +

Google colab pro est entouré comme choix conseillé (?) mais ça ne serait pas plutôt  l'option 1 ?  si je choisi quand même 2, ça ne risque pas de me faire souscrire a un deuxième abonnement, alors que si j'ai bien compris, le choix 1 peur rajouter des unité a mon abonnement actuel  a votre avis  ? 

Hello, off topic question for google colab users:

I have a Google Colab Pro subscription and I have reached the end of my units, I am asked to buy units and I am given 3 choices:

1) - Pay as you go 2) - google colab pro 3) - google colab pro +

Google colab pro is circled as a recommended choice (?) but wouldn't it be option 1 ? if I choose 2 anyway, it won't make me subscribe to a second subscription, while if I understand correctly, choice 1 can add units to my current subscription in your opinion ?

**Comment by TheLastBen** [comment]
for SD, google po (choice 2) is the best choice, but personally, I'd use Runpod or Paperspace if I'm going for paid, the RTX 3090 is 3 times faster and at a good price/speed (0.30$/h)

**Reply by darth2602** [reply]
je suis déja abonné Google Colab Pro , si je prend le choix 2 , je ne vais pas être abonné 2 fois ?   
 le choix 1 ne serait pas mieux juste pour ajouter des unités quand on est deja abonné ?

**Reply by TheLastBen** [reply]
dans ce cas, je ne suis pas sûr

**Reply by darth2602** [reply]
c'ets bien le problème, j'ai demandé a plein de monde et personne n'a su me dire ....
